mythril/src/vmexit.rs

```diff
@@ -233,6 +235,49 @@ impl ExitReason {
     }
 }
 
+#[derive(Clone, Debug, TryFromPrimitive, PartialEq)]
+#[repr(u8)]
+pub enum ApicAccessKind {
+    LinearRead = 0,
+    LinearWrite = 1,
+    LinearFetch = 2,
+    LinearEventDelivery = 3,
+    PhysicalAccessDuringEvent = 10,
+    PhysicalAccessDuringFetch = 15,
+}
+
+#[derive(Clone, Debug)]
+pub struct ApicAccessInformation {
+    pub offset: Option<u16>,
+    pub kind: ApicAccessKind,
+    pub async_instr_exec: bool,
```

Comment:
What is this for?

Reply:
Honestly I'm not really sure. According to Table 27-6: "If the APIC-access VM exit is due to a guest-physical access, this bit is set if the access was asynchronous to instruction execution and not part of event delivery."

I just went ahead and decoded everything in the exit info